bug_actiongroup.php

```diff
@@ -286,6 +286,22 @@
 				$t_failed_ids[$t_bug_id] = lang_get( 'bug_actiongroup_access' );
 			}
 			break;
+		case 'UP_DUE_DATE':
+			$t_due_date = gpc_get_string( 'due_date', null );
+			if( $t_due_date !== null ) {
+				if( is_blank( $t_due_date ) ) {
+					$t_due_date = 1;
```

Comment:
Don't use magic numbers.  Use the following:
     $t_due_date = date_get_null();